doc/all.mk

```diff
@@ -102,7 +102,7 @@ doc/raddb/%.adoc: raddb/%
 
 doc/%.html: doc/%.adoc
 	@echo HTML $^
-	${Q}$(ASCIIDOCTOR) $< -b html5 -o $@ $<
+	${Q}$(ASCIIDOCTOR) $< -b html5 -T ./scripts/asciidoc/freeradius.template -o $@ $<
```

Comment:
This requires the gem `tilt`:

```
asciidoctor: FAILED: 'tilt' could not be loaded
You must have the tilt gem installed (gem install tilt) to use custom backend templates
```

Seems like a good idea, tho.  I would be happier if asciidoc would ignore the argument if the gem wasn't present.

Reply:
the `tilt` could be checked as we do for `asciidoctor` and `pandoc`?

Reply:
Likely best to check at the `configure` stage.

But checking the template, it should really be applied at the PDF stage.   The file `scripts/asciidoc/freeradius.template` is LaTex, and not HTML.

We will likely need a separate HTML / CSS template for the HTML output.  Right now, the colors are red and black, which doesn't match the default FreeRADIUS look.